src/components/rating/Rate.js

```diff
@@ -0,0 +1,49 @@
+/**
+ *  @author sriram
+ *
+ **/
+
+import React, { Component } from 'react';
+import Rating from "react-rating";
+import whitestar from "./star-white.gif";
+import goldstar from "./star-gold.gif";
+import redstar from "./star-red.gif";
+import PropTypes from 'prop-types';
+
+class Rate extends Component {
+    render() {
+        const {label, value, readOnly, total} = this.props;
+        return (
+            <React.Fragment>
+                <b>{label}: </b>
+                <Rating
+                    placeholderRating={value}
+                    emptySymbol={<img src={whitestar} className="icon" alt="white star"/>}
+                    fullSymbol={<img src={goldstar} className="icon" alt="gold star"/>}
+                    placeholderSymbol={<img src={redstar} className="icon"
+                                            alt="red star"/>}
+                    fractions={2}
+                    readonly={true}
+                />
+                <span style={{paddingLeft: 3}}>
+                                ({total})
+                        </span>
```

Comment:
Looks like one of these `span` tags is at the wrong indentation level.

Reply:
Prettier - Please take care of it 😉